### PR TITLE
zh-translation:/docs/ops/traffic-management/protocol-selection/index.md

### PR DESCRIPTION
**Pull request by dk-lockdown**
ref: https://github.com/servicemesher/istio-official-translation/issues/1253

### CODE REVIEW

**Comment by dk-lockdown** [comment]
@googlebot I fixed it.

**Comment by geeknoid** [comment]
To fix the netlify error, you'll need to rebase the PR.

File: content/zh/docs/ops/traffic-management/protocol-selection/index.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Istio 默认支持代理所有 TCP 流量， 但为了提供附加的能力，
 - `udp`
 
 \* 这些协议默认被禁用以阻止偶然地产生试验性的特性。
-要启用他们, 请配置相应的 Pilot [环境变量](/docs/reference/commands/pilot-discovery/#envvars)。
+要启用他们，请配置相应的 Pilot [环境变量](/docs/reference/commands/pilot-discovery/#envvars)。
```
Comment (rootsongjc) [inline content/zh/docs/ops/traffic-management/protocol-selection/index.md]: 请在链接里加上中文地址的前缀 `/zh`。

**Pull request closed without merging**